src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -10475,20 +10479,30 @@ export class StudyViewPageStore
     });
 
     public readonly shouldDisplayPatientTreatments = remoteData({
-        await: () => [this.queriedPhysicalStudyIds],
+        await: () => [this.queriedPhysicalStudyIds, this.unknownQueriedIds],
         invoke: () => {
-            return this.internalClient.getContainsTreatmentDataUsingPOST({
-                studyIds: toJS(this.queriedPhysicalStudyIds.result),
-            });
+            if (this.unknownQueriedIds.result.length === 0) {
+                return this.internalClient.getContainsTreatmentDataUsingPOST({
+                    studyIds: toJS(this.queriedPhysicalStudyIds.result),
+                });
+            } else {
+                return Promise.resolve(false);
+            }
         },
     });
 
     public readonly shouldDisplaySampleTreatments = remoteData({
-        await: () => [this.queriedPhysicalStudyIds],
+        await: () => [this.queriedPhysicalStudyIds, this.unknownQueriedIds],
         invoke: () => {
-            return this.internalClient.getContainsSampleTreatmentDataUsingPOST({
-                studyIds: toJS(this.queriedPhysicalStudyIds.result),
-            });
+            if (this.unknownQueriedIds.result.length === 0) {
```

Comment:
@FlorisVleugels I realized, it would be better to find where these remotes are called and avoid calling them if their are unkonwnQueriedIds.   I.e the check should really be at the page level.    This is true for both shouldDisplayPatientTreatments and shouldDIsplaySampleTreatments.     I.e. it's not really their concert to check for the unkonwnQueriedIds.